src/coreclr/vm/cdac.h

```diff
@@ -0,0 +1,22 @@
+
+#ifndef CDAC_H__
```

Comment:
I'm going to rename this file.  We already have another `cdac.h` in a different directory in #100946 - too confusing.